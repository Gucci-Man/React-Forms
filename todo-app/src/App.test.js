import React from 'react';
import App from './App';
import { render } from "@testing-library/react";

// Smoke test
it("renders without crashing", function () {
  render(<App />);
});

// Snap test
it("matches snapshot", function () {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});