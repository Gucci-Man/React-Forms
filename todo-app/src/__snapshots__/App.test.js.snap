// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <h2>
        Todo List
      </h2>
      <form>
        <label
          for="task"
        >
          Task: 
        </label>
        <input
          id="task"
          name="task"
          placeholder="Your task"
          type="text"
          value=""
        />
        <button>
          Add Todo
        </button>
      </form>
      <div />
    </div>
  </div>
</DocumentFragment>
`;
